app/src/main/java/com/skydoves/disneycompose/utils/NetworkImage.kt

```diff
@@ -38,22 +44,14 @@ fun NetworkImage(
   circularRevealedEnabled: Boolean = false,
   contentScale: ContentScale = ContentScale.Crop
 ) {
-  CoilImage(
-    imageModel = url,
-    modifier = modifier,
+  Image(
+    painter = rememberGlidePainter(url),
+    contentDescription = "",
```

Comment:
I hope to maintain the prior library Landscapist for shimmering and circular revealing effects